design-reference/architecture/aad/easyauth.md

```diff
@@ -0,0 +1,35 @@
+# Purpose/Description
```

Comment:
Can you rename the folder structure to `design-reference/architecture/authn/providers/aad/easyauth.md` so we encourage non azure auth providers?